monitors/SyntheticPegMonitor.js

```diff
@@ -218,45 +218,55 @@ class SyntheticPegMonitor {
     };
   };
 
-  // Takes in two big numbers and returns the error between them.
-  // calculated using: δ = | (observed - expected) / expected |
-  // For example an observed price of 1.2 with an expected price of 1.0 will return | (1.2 - 1.0) / 1.0 | = 0.20
-  // This is equivalent of a 20 percent absolute deviation between the numbers.
+  // Takes in two big numbers and returns the error between them. using: δ = (observed - expected) / expected
+  // For example an observed price of 1.2 with an expected price of 1.0 will return (1.2 - 1.0) / 1.0 = 0.20
+  // This is equivalent of a 20 percent deviation between the numbers.
+  // Note that this logger can return negative error if the deviation is in a negative direction.
   _calculateDeviationError(observedValue, expectedValue) {
```

Comment:
this function no longer returns an absolute value but rather a directional percentage error.